ci/check_spider_naming_consistency.py

```diff
@@ -0,0 +1,262 @@
+import ast
+import os
+import re
+import sys
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+from typing import List, Tuple
+
+COUNTRYCODE_COMPONENTS = {
```

Comment:
Can we just use `pycountry` ISO 3166-1 alpha-2 codes instead?

Reply:
Yea I thought about that but wanted to be pretty restrictive about what got to be capitalized at the end of a spider. I guess that's not a great reason...